CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,112 @@
+  gettsim users.
+- *Being respectful*. We're respectful of others, their positions, their skills, their
+  commitments, and their efforts.
+- *Gracefully accepting constructive criticism*. When we disagree, we are courteous in
+  raising our issues.
+- *Using welcoming and inclusive language*. We're accepting of all who wish to take part
+  in our activities, fostering an environment where anyone can participate and everyone
+  can make a difference.
+
+Our Standards
+-------------
+
+Every member of our community has the right to have their identity respected. The
+gettsim community is dedicated to providing a positive experience for everyone,
+regardless of age, gender identity and expression, sexual orientation, disability,
+physical appearance, body size, ethnicity, nationality, race, or religion (or lack
+thereof), education, or socio-economic status.
+
+Inappropriate Behavior
+~~~~~~~~~~~~~~~~~~~~~~
+
+Examples of unacceptable behavior by participants include:
+
+- Harassment of any participants in any form.
+- Deliberate intimidation, stalking, or following.
+- Logging or taking screenshots of online activity for harassment purposes.
+- Publishing others' private information, such as a physical or electronic address,
+  without explicit permission.
+- Violent threats or language directed against another person.
+- Incitement of violence or harassment towards any individual, including encouraging a
+  person to commit suicide or to engage in self-harm.
+- Creating additional online accounts in order to harass another person or circumvent a
+  ban.
+- Sexual language and imagery in online communities or in any conference venue,
+  including talks.
+- Insults, put downs, or jokes that are based upon stereotypes, that are exclusionary,
+  or that hold others up for ridicule.
+- Excessive swearing.
+- Unwelcome sexual attention or advances.
+- Unwelcome physical contact, including simulated physical contact (eg, textual
+  descriptions like "hug" or "backrub") without consent or after a request to stop.
+- Pattern of inappropriate social contact, such as requesting/assuming inappropriate
+  levels of intimacy with others.
+- Sustained disruption of online community discussions, in-person presentations, or
+  other in-person events.
+- Continued one-on-one communication after requests to cease.
+- Other conduct that is inappropriate for a professional audience including people of
+  many different backgrounds.
+
+Community members asked to stop any inappropriate behavior are expected to comply
+immediately.
+
+Consequences
+------------
+
+If a participant engages in behavior that violates this CoC, the gettsim community CoC
+team may take any action they deem appropriate, including editing/removing inappropriate
+content, warning the offender or expulsion from the community. The CoC team is made up
+of gettsim contributors and users and will react promptly to reports. The current team
+includes
+
+- first
```

Comment:
Is this on purpose?

Reply:
It was included in all other cocs. We can also just state the current gettsim maintainers.